service.go

```diff
@@ -33,11 +34,12 @@ type (
 
 	// CreateServiceRequest are the parameters to create a Service.
 	CreateServiceRequest struct {
-		Cloud       string `json:"cloud,omitempty"`
-		GroupName   string `json:"group_name,omitempty"`
-		Plan        string `json:"plan,omitempty"`
-		ServiceName string `json:"service_name"`
-		ServiceType string `json:"service_type"`
+		Cloud       string                 `json:"cloud,omitempty"`
+		GroupName   string                 `json:"group_name,omitempty"`
+		Plan        string                 `json:"plan,omitempty"`
+		ServiceName string                 `json:"service_name"`
+		ServiceType string                 `json:"service_type"`
+		UserConfig  map[string]interface{} `json:"user_config"`
```

Comment:
It looks like this is an optional field in the API, it'd be great to have this annotated with omitempty on the JSON fields.